reactor-aeron-benchmarks/src/main/java/reactor/aeron/LatencyReporter.java

```diff
@@ -0,0 +1,83 @@
+package reactor.aeron;
+
+import io.scalecube.trace.TraceReporter;
+import java.time.Duration;
+import org.HdrHistogram.Histogram;
+import org.HdrHistogram.Recorder;
+import reactor.core.Disposable;
+import reactor.core.Disposables;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
+public class LatencyReporter {
+
+  private static final TraceReporter reporter = new TraceReporter();
+
+  private final Recorder histogram;
+
+  private final String targetFolder =
+      System.getProperty("reactor.aeron.report.folder.latency", "./target/traces/reports/latency/");
+
+  private String name;
+
+  public LatencyReporter(Recorder histogram, String name) {
+    this.histogram = histogram;
+    this.name = name;
+  }
+
+  public Disposable start() {
+    return Disposables.composite(startReport(), startCollect());
+  }
+
+  
+  private Disposable startCollect() {
+    return Flux.interval(
+            Duration.ofSeconds(Configurations.WARMUP_REPORT_DELAY),
+            Duration.ofSeconds(Configurations.REPORT_INTERVAL))
+        .publishOn(Schedulers.single())
+        .doOnNext(i -> this.collect())
+        .subscribe();
+  }
+
+  private Disposable startReport() {
+    return startReport(targetFolder);
+  }
+
+  private Disposable startReport(String folder) {
+    return Flux.interval(
+            Duration.ofSeconds(Configurations.WARMUP_REPORT_DELAY),
+            Duration.ofSeconds(Configurations.REPORT_INTERVAL + 30))
```

Comment:
let's don't use a magic number like 30 - it will be better to wrap it as a property in `reactor.aeron.Configurations`.